Makefile.in

```diff
@@ -20,6 +20,9 @@ build: euimage
 
 build/image/centos-7-builder.raw: centos-7-builder.ks
 	mkdir -p build/image
+	sed -i.bak 's/^#user = "root"/user = "root"/' /etc/libvirt/qemu.conf
+	sed -i.bak 's/^#group = "root"/group = "root"/' /etc/libvirt/qemu.conf
+	systemctl restart libvirtd.service
```

Comment:
Yeah, this is going to prevent make from running as anyone other than root.  At most this step, along with an explanation of what _specifically_ it fixes and why, should go in the README file.